apps/src/code-studio/components/progress/ScriptTeacherPanel.jsx

```diff
@@ -42,19 +61,21 @@ class ScriptTeacherPanel extends React.Component {
       sectionsAreLoaded,
       scriptHasLockableStages,
       scriptAllowsHiddenStages,
-      unlockedStageNames
+      unlockedStageNames,
+      students
     } = this.props;
+    const studentTableMaxHeight = this.calculateStudentTableMaxHeight();
 
     return (
       <TeacherPanel>
-        <h3>{commonMsg.teacherPanel()}</h3>
-        <div className="content">
+        <div id="nonscrollable-content">
```

Comment:
this id is a bit generic and risks eventual collision with other ids in the DOM. how about just `teacher-panel-content`? or if you want it to still be more descriptive, `teacher-panel-nonscrollable-content` or `teacher-panel-nonscrollable`?